config/jest/setupTests.ts

```diff
@@ -16,3 +16,16 @@ window.crypto.randomUUID = (() => {
     return value.toString();
   };
 })();
+
+// Mock focus-trap-react to prevent errors in tests that use modals. focus-trap-react is a package used
+// by LeafyGreen and is not a direct dependency of this repo.
+jest.mock(
```

Comment:
If I had to guess it's probably because the underlying renderer changed from React 17 to React 18 and we're on older LeafyGreen components that only officially support React 17